test/embag_test_python.py

```diff
@@ -178,5 +178,11 @@ def testBufferInfo(self):
             for covariance in np.array(covariance_array, copy=False):
                 self.assertEqual(covariance, 0)
 
+    def testToDictMemoryView(self):
+        for msg in self.view.getMessages('/base_pose_ground_truth'):
+            dict_list = msg.dict()['pose']['covariance'].tolist()
+            data_list = [v for v in msg.data()['pose']['covariance']]
+            assert dict_list == data_list
+
```

Comment:
How does this test actually check the memory view?

Reply:
This essentially attempts to validate that the `tolist()` from the memory view is equivalent to iterating through each of the values in the list via the RosValue interface.